### DISCUSSION TITLE
calcGrains - packets

### DISCUSSION THREAD

**Discussion by ArianeNeves** [mtex-toolbox/mtex#1219] [Ask Anything]
Hi!

Please, is it possible to use the two conditions below to identify the packets boundaries in a martensite microstructure?

packetsGrains = parentEBSD.calcGrains('custom',packetId,'delta',1);

[parentGrains,parentEBSD.grainId] = calcGrains(parentEBSD,'unitCell','angle',15*degree);

Thanks in advance!

**Comment by nyyssont** [comment]
As far as I know, the commands above don't work. I think there might be a variant-based grain construction routine coming up in a future release, but until then one should use the merge command following grain pairs that have a similar packetId and parent grain Id as in this prior discussion:

https://github.com/mtex-toolbox/mtex/discussions/1005
